lib/style-compiler/index.js

```diff
@@ -9,10 +9,10 @@ module.exports = function (css, map) {
   this.cacheable()
   var cb = this.async()
 
-  var query = loaderUtils.getOptions(this) || {}
-  var vueOptions = this.options.__vueOptions__
+  var query = Object.assign({}, loaderUtils.getOptions(this), this.options.vue, this.vue)
 
-  if (!vueOptions) {
+  // Slightly messy check for HappyPack as we can't use this.options.__vueoptions__ anymore
+  if (this._compiler.constructor.name === 'HappyFakeCompiler') {
```

Comment:
Hmm, I cannot review this line due to I'm not very familiar with `HappyPack`. 🤔 
/ping @yyx990803